copyblogger/genesis-simple-hooks
# Add a header to prevent a fatal error

Fixes an issue a user raised to support: that this sometimes causes a fatal error on installing via `wp-admin/plugin-install.php`

### How to test

1. Go to `/wp-admin/plugin-install.php`
2. Search for Genesis Simple Hooks
<img width="1027" alt="Screenshot 2023-09-29 at 12 14 40 PM" src="https://github.com/studiopress/genesis-simple-hooks/assets/4063887/87eb3e85-6ef0-48ce-8a9e-d0891809648b">


3. Install and activate it
4. Expected: The page loads
5. Actual: There's sometimes an error, though you might not see it:

<img width="1066" alt="Screenshot 2023-09-29 at 12 13 31 PM" src="https://github.com/studiopress/genesis-simple-hooks/assets/4063887/952322cc-6509-447a-8d05-03879338739b">

That's because the header [was missing](https://github.com/studiopress/genesis-simple-hooks/pull/37#discussion_r1341831830).
